core/drivers/crypto/caam/include/caam_desc_helper.h

```diff
@@ -238,7 +238,8 @@ static inline void dump_desc(uint32_t *desc)
  * RNG Generates Data
  */
 #define RNG_GEN_DATA                                                           \
-	(CMD_OP_TYPE | OP_TYPE(CLASS1) | OP_ALGO(RNG) | ALGO_AS(RNG_GENERATE))
+	(CMD_OP_TYPE | OP_TYPE(CLASS1) | OP_ALGO(RNG) | ALGO_AS(RNG_GENERATE) |\
```

Comment:
Could you align `\` ?

Reply:
Hm, this is aligned in my editor, as are the other defines. I think this may be a github display issue.